model/registry/file.yaml

```diff
@@ -4,12 +4,56 @@ groups:
     type: attribute_group
     brief: "Describes file attributes."
     attributes:
+      - id: accessed
+        type: string
+        brief: >
+          Time when the file was last accessed, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems.
```

Comment:
What exact specifics you want to have for this field?

This field might not be present for some operation systems or not behave as one would expect as many filesystems disable updating the atime for performance reasons, but I would be against putting here concrete examples as defaults might change in the future. Still I want to have this note present. Do you want me to rephrase it more clearly?

```suggestion
          This attribute might not be supported by some operating systems.
```

Reply:
The specifics I'd like to see is which OS attribute is available on. Since there is no semconv that references this attribute, it may or may not be available on any telemetry item and saying that it's not  available sometimes does not add any information.

Reply:
>> I would be against putting here concrete examples as defaults might change in the future

> is which OS attribute is available on

I don't want to put here enumeration. There are many filesystems (i.e. in embedded OS, NSF, FAT32 etc) which do not maintain access time or update it consistently due to performance reasons. 

> it's not available sometimes does not add any information.

I think it adds more value to the enduser to have this note. Because it is not `not available sometimes`. It either available for that specific OS/FS or not available. And if I gather telemetry from different endpoints it's good to know that this field might be empty due to restrictions of OS/FS itself

Reply:
I have updated comment for this one and the following similar attribute, could you check if it looks better now